src/Substructure.jl

```diff
@@ -116,12 +53,13 @@ The method stops at the first jet satisfying the mass and distance thresholds.
 # Arguments:
 - `jet::PseudoJet`: PseudoJet instance representing the jet to tag.
 - `clusterseq::ClusterSequence`: ClusterSequence with jet clustering history.
-- `tag::MassDropTagger`: MassDropTagger instance providing mass drop parameters.
+- `mu::Float64`: Maximum allowed mass ratio for a jet to pass tagging.
+- `y::Float64`: Minimum kT distance threshold for parent separation.
 
 # Returns:
 - `PseudoJet`: The jet (or subjet) satisfying the mass drop conditions, if tagging is successful, otherwise a zero-momentum PseudoJet
 """
-function mass_drop(jet::PseudoJet, clusterseq::ClusterSequence, tag::MassDropTagger)
+function mass_drop(jet::PseudoJet, clusterseq::ClusterSequence; mu::Float64, y::Float64)
```

Comment:
```suggestion
function mass_drop(jet::PseudoJet, clusterseq::ClusterSequence{PseudoJet}; mu::Float64, y::Float64)
```

could we use this opportunity to also constrain `ClusterSequence` parameter here and and other substructure functions?